ServerApp/app/__init__.py

```diff
@@ -6,23 +6,45 @@
 import subprocess
 import time
 import re
+import json
+import pickle
 
 from datetime import datetime
 from datetime import timedelta
-from flask import Flask
+from flask import *
 from flask import jsonify
 from flask import request
 from flask_cors import CORS
+from flask_sqlalchemy import SQLAlchemy
+from functools import wraps
 
 app = Flask(__name__)
 CORS(app)
+app.config.from_object('config')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(app)
 
 diskReadNew = 0
 diskReadOld = 0
 diskWriteNew = 0
 diskWriteOld = 0
 processid = False
 
+from app.user.controllers import mod_user, is_logged_in
+app.register_blueprint(mod_user)
+
+def check_auth(request):
```

Comment:
Documentation is  missing. Make sure to put proper documentation. Use PEP8 standards